extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php

```diff
@@ -47,16 +47,14 @@ public function getCentralUrlForLang( $lang ) {
 	 * @return bool|GlobalTitle
 	 */
 	public function getCentralWikiUrlForLangIfExists( $lang ) {
```

Comment:
Does this method still return anything?  I don't think we have corporate wikis. ?

Reply:
Unfortunately, yes - we still have them. Their domains are used to link to WordPress-powered sites.